blog/2024-03-08-structuring-a-large-app.md

```diff
@@ -0,0 +1,391 @@
+---
+author: Masen Furer
+date: 2024-03-08
+title: "Structuring a Large App"
+description: "So your Reflex app is getting large? Here's some advice on how to lay it out."
+image: /blog/structure-large-app.jpg
+---
+
+# First Steps
+
+Reflex makes it easy to get a simple web app running from a single python file:
+
+```python
+import reflex as rx
+
+
+@rx.page()
+def index():
+    return rx.center(
+        rx.heading("Hello Reflex! 👋")
+    )
+
+
+app = rx.App()
+```
+
+However, as the app grows, keeping everything in a single module can quickly
+become messy and unmanageable. Because a Reflex app is just a python project,
+it can be organized into modules and packages (directory with `__init__.py`)
+and use the python import machinery you're likely already familiar with.
```

Comment:
style: machinery may be a scary word, maybe reword to "imported like any other python object"